src/Draco.LanguageServer/Translator.cs

```diff
@@ -62,4 +64,80 @@ public static LspModels.Position ToLsp(CompilerApi.Syntax.SyntaxPosition positio
         Line = (uint)position.Line,
         Character = (uint)position.Column,
     };
+
+    public static LspModels.CompletionItem ToLsp(CompilerApi.CodeCompletion.CompletionItem item)
+    {
+        var textEdit = ToLsp(item.Edits[0]);
+        var additionalEdits = item.Edits.Skip(1).Select(ToLsp).ToList();
+
+        var detail = string.Empty;
+        if (item.Symbols.FirstOrDefault() is CompilerApi.Semantics.ITypedSymbol typed)
+        {
+            detail = item.Symbols.Count() == 1 ? typed.Type.ToString() : $"{item.Symbols.Count()} overloads";
```

Comment:
Please use `.Length` at both places, `Count()` is a LINQ method that potentially iterates the collection to count each element